spec/matchers/permit_actions_spec.rb

```diff
@@ -1,6 +1,25 @@
 require 'rspec/core'
 
 describe 'permit_actions matcher' do
+  before { allow(::Kernel).to receive(:warn) }
```

Comment:
Admittedly, this is also sort of hacky. This is just to prevent warnings when running the test suite, which is probably a matter of preference?